### ISSUE TITLE
Failed to load config/bundles.php with symfony 5.1 app

### ISSUE THREAD

**Issue by magnetik** [box-project/box#503]
## Bug report

| Question    | Answer
| ------------| ---------------
| Box version | 3.9.1
| PHP version | 7.4.10
| Platform with version | Ubuntu
| Github Repo | -

The documentation states that it should work out of the box, and it did with the minor changes.

But with my symfony 5.2 app, when trying to execute the application : `failed to open stream: phar error: "src/config/bundles.php" is not a file in phar`

The file bundles.php is present in the phar, in the correct place:

```
> % box info /usr/local/bin/clusterctl --list  | grep -C 2 bundles.php
  console [NONE] - 1.24KB
config/
  bundles.php [NONE] - 97.00B
  packages/
    cache.yaml [NONE] - 687.00B
```

For some reason, it seems that the MicroKernelTrait from Symfony : https://github.com/symfony/symfony/blob/5.1/src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php#L71
load bundles.php from the ProjectDir (ie. the directory of the composer.json) 

getProjectDir is defined in the Kernel as:  https://github.com/symfony/symfony/blob/5.x/src/Symfony/Component/HttpKernel/Kernel.php#L279-L299

The project dir is detected by stopping going up the directory tree when composer.json is found.

**TL;DR: Symfony detects the project dir as the folder where composer.json is, because it's deleted by box, symfony 5.1 app won't work out of the box**

**Comment by theofidry** [comment]
Hm indeed and this is confirmed by our tests: https://github.com/box-project/box/blob/master/fixtures/build/dir012/src/Kernel.php#L28

Could you mention this in the Symfony support doc entry?
